diff -r 828006bb0a98 examples/CMakeLists.txt
--- a/examples/CMakeLists.txt   Mon Apr 16 21:10:28 2018 +0200
+++ b/examples/CMakeLists.txt   Wed Apr 18 09:54:40 2018 -0700
@@ -106,7 +106,7 @@
     "${CMAKE_CURRENT_BINARY_DIR}/${TEST_NAME}_fail.xml"
     @ONLY)
   set(_env_vars)
-  list(APPEND _env_vars "PKG_CONFIG_PATH=${example_INSTALL_DIR}/lib/pkgconfig:$PKG_CONFIG_PATH")
+  list(APPEND _env_vars "PKG_CONFIG_PATH=${example_INSTALL_DIR}/${CMAKE_INSTALL_LIBDIR}/pkgconfig:$PKG_CONFIG_PATH")
   add_test(${TEST_NAME}
     ${CMAKE_CURRENT_SOURCE_DIR}/test_core_child_requires_core_no_deps.bash
   )
